run.sh

```diff
@@ -1,20 +1,5 @@
 #!/bin/sh
 
 set -x
-
-# This is the entrypoint for the image and meant to wrap the
-# logic of gathering/reporting results to the Sonobuoy worker.
-
-results_dir="${RESULTS_DIR:-/tmp/results}"
-
-# saveResults prepares the results for handoff to the Sonobuoy worker.
-# See: https://github.com/vmware-tanzu/sonobuoy/blob/master/docs/plugins.md
-saveResults() {
-	# Signal to the worker that we are done and where to find the results.
-	printf "${results_dir}"/out.json > "${results_dir}"/done
```

Comment:
The main output file is now the `sonobuoy_results.yaml`. We could still do the trap here but:
 - the writing of the done file is now done in go code
 - if we keep the trap it should just be the whole directory, not just the output file (we send both the out.json and the sonobuoy_results.yaml file)

Reply:
The only way this is necessary is if we are concerned about panics. Up to you if you want me to add it back; but regardless of pass/fail, the donefile gets written.